### ISSUE TITLE
kemal return 200 status instead of 404 (after update to 0.25)

### ISSUE THREAD

**Issue by kostya** [kemalcr/kemal#509]
after update kemal, i find this bug, when request unknown path.
kemal 0.25.1, Crystal 0.27.0

```crystal
require "kemal"

get "/" do
  "Hello World!"
end

Kemal.run
```

```
curl -I http://127.0.0.1:3000/bla
HTTP/1.1 200 OK
Connection: keep-alive
X-Powered-By: Kemal
Content-Type: text/html
Transfer-Encoding: chunked
```

**Comment by sdogruyol** [comment]
Which Kemal version were you using before updating to `0.25.1` because this is also happening in `0.25.0` :thinking:

**Comment by kostya** [comment]
it was 0.24.0

**Comment by mamantoha** [comment]
Hi @sdogruyol . Check this https://github.com/kemalcr/kemal/pull/513

**Comment by sdogruyol** [comment]
Fixed via #513
